### ISSUE TITLE
Methods `catchPhrase()` and `bs()` are missing as `@method` declaration

### ISSUE THREAD

**Issue by kevinpapst** [FakerPHP/Faker#621]
### Summary

Running PHPStan on a changed bit of code leads from `$faker->catchPhrase is deprecated` to `$faker->catchPhrase() is an unknown method`.

### Versions

|                | Version |
|:---------------|:--------|
| PHP            | -   |
| `fakerphp/faker`  | 1.21.0   |

### Self-enclosed code snippet for reproduction

```php
$faker = Factory::create();
$faker->catchPhrase();
```

The folowing snippet needs to be added to the huge Generator docblock at
https://github.com/FakerPHP/Faker/blob/main/src/Faker/Generator.php#L8

```
 * @property string $catchPhrase
 *
 * @method string catchPhrase()
 * 
 * @property string $bs
 *
 * @method string bs()
```

**Comment by pimjansen** [comment]
We only added the main methods and for every single locale though since this causes other sidefx.

**Comment by kevinpapst** [comment]
> the main methods

To me these two methods are used in 80% of my fixtures. While I do not use 90% of the annotated methods. So "main" methods sounds a bit arbitrary to me 😁 

Does that mean you do not plan to add them? 
Or does that mean you do not accept PRs adding them?
I'd love to reduce my PHPStan baseline...

**Comment by kevinpapst** [comment]
Question is still open @pimjansen

**Comment by pimjansen** [comment]
> Question is still open @pimjansen

Sorry for the late reply. If they are not in the generic settings and are locale specific we decided to not add them.

At the design for Faker 2.x we removed all the magic anyway where  you will just hit an interface or implementation directly

**Comment by kevinpapst** [comment]
Thanks for your answer 👍 

Are you talking about 2x, which received the last commit a year ago: https://github.com/FakerPHP/Faker/commits/2.x ?
From my experience, such branches are dead and won't ever be published. 
Or are you talking about something different?

**Comment by pimjansen** [comment]
Ah no there is work done and the global interfaces are generated including generic methods (see faker core).

The port of the english locale should still be done for example

**Comment by kevinpapst** [comment]
Okay, just to summarize: this issue will be solved with 2.0, while fixing it in the current live version is not planned.

Then I will close it and wait for future builds. Thanks for your feedback!
